app/src/Components/Bubbles/Bubbles.css

```diff
@@ -0,0 +1,65 @@
+.App {
```

Comment:
We should define how to style the elements. Keep classname, use CSS-in-JS or styled components?